www/content/workstation/chef_run.md

```diff
@@ -119,21 +119,28 @@ chef-run host1 my_cookbook::non_default_recipe
 ```
 
 `::recipe_name` tells `chef-run` to run a recipe other than the default.
-`chef-run` reads your local `~/.chef/config.rb` and looks for cookbooks in the
-paths specified as `cookbook_path`. That configuration value is an array and
-looks something like this:
+`chef-run` reads your local Chef Workstation configuration file `~/.chef-workstation/config.toml` and Chef configuration file `~/.chef/config.rb`. It looks for cookbooks in the paths specified in both files. The configuration value is an array and looks something like this:
+
+For workstation configuration file:
+
+```ruby
+[chef]
+cookbook_repo_paths = [
+  "/path/1",
+  "/path/b"
+]
+```
+
+and for Chef configuration file:
```

Comment:
Suggest "`~/.chef/config.rb`:" instead of "for Chef configuration file"

Reply:
changed.